packages/react-menu/src/stories/MenuCustomInnerFunctionTrigger.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { ChevronDown16Regular } from '@fluentui/react-icons';
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Menu, MenuTrigger, MenuList, MenuItem, MenuPopover, MenuTriggerChildProps } from '@fluentui/react-menu';
+
+// FIXME need to redeclare types since type import is under a @ts-ignore
+type MenuOpenEvents =
+  | MouseEvent
+  | TouchEvent
+  | React.MouseEvent<HTMLElement>
+  | React.KeyboardEvent<HTMLElement>
+  | React.FocusEvent<HTMLElement>;
+
+const buttonStyle = { height: 22, verticalAlign: 'middle' };
+
+export const CustomInnerFunctionTrigger = () => {
```

Comment:
I would rename it to include render function, since that's the accepted term I've seen in similar examples

```suggestion
export const RenderFunctionTrigger= () => {
```